targets/TARGET_ARM_SSG/TARGET_MUSCA_A1/device/COMPONENT_SPE/TOOLCHAIN_ARMC6/musca_s.sct

```diff
@@ -0,0 +1,211 @@
+#! armclang --target=arm-arm-none-eabi -march=armv8-m.main -E -xc
+
+/*
+ * Copyright (c) 2018 ARM Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "../../../partition/region_defs.h"
+
+#if !defined(TFM_LVL)
+  #define TFM_LVL   1
+#endif
+
+LR_CODE S_CODE_START {
+
+    /****  This initial section contains common code for TEE */
+    ER_TFM_CODE S_CODE_START S_CODE_SIZE {
+        *.o (RESET +First)
+        .ANY (+RO)
+    }
+
+#if TFM_LVL == 1
+
+    ER_TFM_DATA S_DATA_START S_DATA_SIZE {
+        .ANY (+RW +ZI)
+    }
+
+    TFM_SECURE_STACK +0 ALIGN 128 EMPTY 0x2000 {
+    }
+
+    TFM_UNPRIV_SCRATCH +0 ALIGN 32 EMPTY 0x400 {
+    }
+
+#else /* TFM_LVL == 1 */
+
+    /**** Unprivileged Secure code start here */
+    TFM_UNPRIV_CODE +0 ALIGN 32 {
+        tfm_unpriv_api.o (+RO)
+        dummy_crypto_keys.o (+RO)
+        dummy_nv_counters.o (+RO)
+        platform_retarget_dev.o (+RO)
+        *(SFN)
+        *armlib*
+    }
+
+    TFM_SP_STORAGE +0 ALIGN 32 {
+        *tfm_storage* (+RO)
+        *(TFM_SP_STORAGE_ATTR_FN)
+    }
+
+    TFM_SP_AUDIT_LOG +0 ALIGN 32 {
```

Comment:
Since audit log and storage will not be part of Mbed-OS, linker script should be updated as part of patch to remove these features

Reply:
You are right, the LD files are generated in TF-M but the SCT files are not, need to manually change them.

Reply:
Done